packages/account/src/Sections/Security/Passwords/deriv-email.tsx

```diff
@@ -1,64 +1,64 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import { Formik } from 'formik';
-import { withRouter } from 'react-router';
-import { WS, toTitleCase } from '@deriv/shared';
+import { toTitleCase } from '@deriv/shared';
 import { Localize, localize } from '@deriv/translations';
 import { Button, Text, Input } from '@deriv/components';
 import FormSubHeader from 'Components/form-sub-header';
 import SentEmailModal from 'Components/sent-email-modal';
 import UnlinkAccountModal from 'Components/unlink-account-modal';
 import { observer, useStore } from '@deriv/stores';
+import { useVerifyEmail } from '@deriv/hooks';
 
-const DerivEmail = observer(({ email, social_identity_provider, is_social_signup }) => {
-    const { common } = useStore();
-    const { is_from_derivgo } = common;
+type TDerivEmailProps = {
+    email: string;
+    social_identity_provider: string;
+    is_social_signup: boolean;
+};
+
+const DerivEmail = observer(({ email, social_identity_provider, is_social_signup }: TDerivEmailProps) => {
+    const {
+        common: { is_from_derivgo },
+    } = useStore();
+    const { send } = useVerifyEmail('request_email');
     const [is_unlink_account_modal_open, setIsUnlinkAccountModalOpen] = React.useState(false);
     const [is_send_email_modal_open, setIsSendEmailModalOpen] = React.useState(false);
 
     const onClickChangeEmail = () => {
         if (is_social_signup) {
             setIsUnlinkAccountModalOpen(true);
         } else {
-            WS.verifyEmail(email, 'request_email');
+            send(email);
             setIsSendEmailModalOpen(true);
         }
     };
 
     const onClickSendEmail = () => {
-        WS.verifyEmail(email, 'request_email');
+        send(email);
         setIsUnlinkAccountModalOpen(false);
         setIsSendEmailModalOpen(true);
     };
 
-    const onClickResendEmail = () => {
-        WS.verifyEmail(email, 'request_email');
-    };
-
     return (
         <React.Fragment>
             <FormSubHeader title={localize('Email address')} />
             <div className='account__email-wrapper'>
-                <React.Fragment>
-                    <Text as='p' className='email-platform__desc' color='prominent' size='xs' weight='lighter'>
-                        <Localize i18n_default_text='This is the email address associated with your Deriv account.' />
-                    </Text>
-                </React.Fragment>
+                <Text as='p' className='email-platform__desc' color='prominent' size='xs' weight='lighter'>
+                    <Localize i18n_default_text='This is the email address associated with your Deriv account.' />
+                </Text>
                 <div className='email-platform__content'>
-                    <Formik>
+                    <form>
```

Comment:
Do we really need form if we're not using anything related to it?

Reply:
Yes. That is why I removed Formik. But kept form for HTML semantic purpose. Any suggestions?